src/pages/Map.css

```diff
@@ -0,0 +1,70 @@
+.PinCard {
+  width: 250px;
+  height: 200px;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-around;
+}
+
+.PinCardLabel {
+  width: max-content;
+  color: pink;
+  font-size: 15px;
+  border-bottom: 1px solid pink;
+  margin: 3px 0;
```

Comment:
Same, `rem` is a better practice for margins. 
`em` is a better practice for paddings